DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditCourseDetails.cshtml

```diff
@@ -0,0 +1,172 @@
+        <fieldset class="nhsuk-fieldset">
+          <div class="nhsuk-checkboxes nhsuk-checkboxes--conditional">
+
+            <div class="nhsuk-u-margin-bottom-3">
+              <div class="nhsuk-checkboxes__item">
+                <input class="nhsuk-checkboxes__input"
+                       asp-for="PasswordProtected"
+                       type="checkbox"
+                       aria-controls="@passwordId"
+                       aria-expanded="@(Model.PasswordProtected.ToString())">
+                <label class="nhsuk-label nhsuk-checkboxes__label" asp-for="PasswordProtected">
+                  Password protected
+                </label>
+              </div>
+
+              <div class="@passwordCss" id="@passwordId">
+                <vc:text-input asp-for="@nameof(Model.Password)"
+                               label="Enter the password for the course"
+                               populate-with-current-value="true"
+                               type="text"
+                               spell-check="false"
+                               autocomplete="off"
+                               hint-text=""
+                               css-class="nhsuk-u-width-one-half" />
+              </div>
+            </div>
+
+            <div class="nhsuk-checkboxes__item">
+              <input class="nhsuk-checkboxes__input"
+                     asp-for="ReceiveNotificationEmails"
+                     type="checkbox"
+                     aria-controls="@emailId"
+                     aria-expanded="@(Model.ReceiveNotificationEmails.ToString())">
+              <label class="nhsuk-label nhsuk-checkboxes__label" asp-for="ReceiveNotificationEmails">
+                Receive course completion notification emails
+              </label>
+            </div>
+
+            <div class="@emailCss" id="@emailId">
+              <vc:text-input asp-for="@nameof(Model.NotificationEmails)"
+                             label="Enter an email address for receiving the notifications"
+                             populate-with-current-value="true"
+                             type="text"
+                             spell-check="false"
+                             autocomplete="off"
+                             hint-text=""
+                             css-class="nhsuk-u-width-one-half" />
+            </div>
+
+          </div>
+        </fieldset>
+      </div>
+
+      <h2 class="nhsuk-heading-s nhsuk-u-font-weight-normal">
+        Completion criteria
+      </h2>
+
+      <div class="nhsuk-form-group">
+        <fieldset class="nhsuk-fieldset">
+          <div class="nhsuk-radios nhsuk-radios--conditional">
+
+            @if (Model.PostLearningAssessment) {
+              <div class="nhsuk-radios__item nhsuk-u-margin-bottom-3">
+                <input class="nhsuk-radios__input"
+                       id="PostLearningAssessmentRadio"
+                       name="IsAssessed"

```

Comment:
I think you should still be able to get rid of the name attribute both here and in the below radio

Reply:
removed :)